.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:4000
```

Comment:
ENV variable - there are a few fetches currently that are NOT using this and we will need to change that, but the main fetch is working with this variable.